static/js/types.ts

```diff
@@ -0,0 +1,15 @@
+// TODO/typescript: Move this to server_events_dispatch
+export type UserGroupUpdateEvent = {
+    id: number;
+    type: string;
+    group_id: number;
+    data: {
+        name?: string;
+        description?: string;
+    };
+};
+
+// TODO/typescript: Move the User and Stream placeholder
+// types to their approriate modules.
+export type User = Record<string, unknown>;
+export type Stream = Record<string, unknown>;
```

Comment:
These should be stubbed as `never` until we have something to put here, again so we don’t unsoundly allow `is_user_group({members: "foo"})`.

Reply:
Oops, fixed!